h3/src/server.rs

```diff
@@ -118,13 +138,25 @@ where
     C: quic::Connection<B>,
     B: Buf,
 {
-    /// Create a new HTTP/3 server connection with default settings
+    /// Create a new HTTP/3 server connection
     ///
-    /// Use a custom [`Builder`] with [`builder()`] to create a connection
-    /// with different settings.
-    /// Provide a Connection which implements [`quic::Connection`].
-    pub async fn new(conn: C) -> Result<Self, Error> {
-        builder().build(conn).await
+    /// Provide a connection endpoint which implements [`quic::Connection`] and
+    /// desired configuration for the server.
+    pub async fn new(conn: C, config: ServerConfig) -> Result<Self, Error> {
```

Comment:
There is this `new` method and the `new` function in line 97. 
The `new` function just calls this method.

This `new` method is also public so there are two ways possible to create a server `Connection`
I would remove this `new` method and put the code also in the new function like it is in the client.

Reply:
I guess there is no harm of keeping this right?

Reply:
Yes probably not